README.md

```diff
@@ -71,6 +71,27 @@ custom:
     endpointType: 'regional'
 ```
 
+Alternatively, you can configure the plugin to create domains for both REST and websocket APIs (example for `serverless.foo.com/api`).
+
+```yaml
+custom:
+  customDomain:
+    domainName: serverless.foo.com
+    stage: ci
+    basePath: api
+    certificateName: '*.foo.com'
+    createRoute53Record: true
+    endpointType: 'regional'
+    websockets:
```

Comment:
Is there a way that we could just have `customDomain` take in a list of domain name configurations rather than have a separate config section just for websockets? Could also help with closing #88 if implemented correctly, too.

Reply:
This is surely a more general solution. Smth like that will be backward compatible if the plugin assumes `websocket: false` when omitted in the config:

```yaml
custom:
  customDomain:
     - domainName: serverless.foo.com
        stage: ci
        basePath: api
        certificateName: '*.foo.com'
        createRoute53Record: true
        endpointType: 'regional'
     - domainName: serverless-ws.foo.com
        stage: ci
        certificateName: '*.foo.com'
        createRoute53Record: true
        endpointType: 'regional'
        websocket: true
```

What do you think?

Reply:
Yep, this is along the lines of we were thinking.